zio-sql/jvm/src/main/scala/Sql.scala

```diff
@@ -185,11 +190,23 @@ trait Sql {
 
     type Aux[A] = Table { type TableType = A }
 
-    sealed trait Source[F[_], A] extends Table {
+    sealed trait Source extends Table {
+      type Repr[_]
+      type Cols
       val name: TableName
-      val columnSchema: ColumnSchema[A]
-      val columns: F[TableType]
-      val columnsUntyped: List[Column.Untyped]
+      val columnSchema: ColumnSchema[Cols]
+      val columns: Repr[TableType]
+    }
+    object Source {
+      type Aux_[F[_], B] = Table.Source {
+        type Repr[X] = F[X]
+        type Cols    = B
+      }
+      type Aux[F[_], A, B] = Table.Source {
```

Comment:
This one is not used, so we can delete it, and rename `Aux_` => `Aux`.

Reply:
@jdegoes `deleteFrom` (line 238) uses `Aux[F[_], A, B]` because it needs to know what `TableType` (aka `A`) is to pass through DeleteBuilder to ensure the `where` expression is valid.
`table` (line 104) uses `Aux_[F[_], B]` because it can't know what `TableType` (aka `A`) is for the return type.
Both are only used once, but I can't see a way to use the same `Aux` pattern for both.